setup.py

```diff
@@ -74,6 +74,9 @@
     "pynacl==1.3.0",
     "dataclasses>=0.7, <1;python_version<'3.7'",
     "async_generator==1.10",
+    "trio>=0.13.0",
```

Comment:
Missing `trio_typing`

Reply:
Could you point out where we require `trio_typing`? I'm not able to find it used in the codebase.

Reply:
Oh, please check here. https://github.com/libp2p/py-libp2p/pull/396/files#r369460417

Reply:
Nice catch! Fixed in https://github.com/libp2p/py-libp2p/pull/365/commits/e57d01f360de9fb101adffa686cf5aff414fc8f8.